README.md

```diff
@@ -75,8 +75,8 @@ None.
 Here's how to use it in a playbook:
 
 ```yaml
-- hosts: teamservers
-  become: yes
+- hosts: all
```

Comment:
Do we care about the change here?

Reply:
No, we do not.  It's just example code, and by changing to match the skeleton we avoid this conflict in the future.